dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/metadata/ConfigurableMetadataServiceExporter.java

```diff
@@ -113,7 +117,8 @@ private ProtocolConfig generateMetadataProtocol() {
 
         ProtocolConfig protocolConfig = new ProtocolConfig();
         protocolConfig.setName(specifiedProtocol);
-
+        String serialization = getSerialization(specifiedProtocol);
+        protocolConfig.setSerialization(serialization);
```

Comment:
Maybe we can merge all of the config from exist protocol config, or directly reuse(if user do not specify different port).